### PR TITLE
Fix a confusing React component example

### PR DESCRIPTION
**Pull request by Kirshach**
#### Description (required)

I believe the navigation triggering React component has a just very slightly confusing name and an unused prop which somewhat misguide the reader's attention

#### Related issues & labels (optional)

- Suggested label: Fix a confusing React component example

### CODE REVIEW

File: src/content/docs/en/guides/view-transitions.mdx [hunk 1]
```diff
@@ -309,7 +309,7 @@ The following example implements the same using `navigate()` in a React `<Form /
 import React from 'react';
 import { navigate } from 'astro:transitions/client';
 
-export default function ClickToNavigate({ to }) {
+export default function Form() {
```
Comment (at-the-vr) [inline src/content/docs/en/guides/view-transitions.mdx:312]: Missing `to` prop ⬇️ 
```suggestion
export default function Form({ to }) {
```

**Review by at-the-vr — COMMENTED** [review]
Hi there @Kirshach `ClickToNavigate` suggests readers that this function is meant to well "navigate to different page on click". Changing its name to plain `Form` semantically raises questions eg: what's Form supposed to do ? any action related with it (submit, fetch) ? 

But importantly at [Line 325](https://github.com/withastro/docs/blob/main/src/content/docs/en/guides/view-transitions.mdx#L325) there's the same thing your thinking off

That being said I only have 1 suggestion

File: src/content/docs/en/guides/view-transitions.mdx (hunk 1 above)
Comment (Kirshach) [reply src/content/docs/en/guides/view-transitions.mdx:312]: Hi! Thank you for your reply!
Why would we include a "to" prop and not use it? 🤔

Comment (at-the-vr) [reply src/content/docs/en/guides/view-transitions.mdx:312]: your right, `to` can be removed 😄 , suggestion cancelled

**Review by sarah11918 — APPROVED** [review]
Agreed! This is a great change. Thank you for taking the time to submit! We really appreciate it, and are happy to welcome you to Team Docs!

**Pull request merged**